### ISSUE TITLE
Compile error after applying format in generic type of UnsafePointer

### ISSUE THREAD

**Issue by longkai** [nicklockwood/SwiftFormat#82]
After applying `swiftformat .`, ran `git diff` and got,

```sh
-    return line.withUnsafeBytes { (bytes: UnsafePointer<Int8>) -> (String, Int)? in
+    return line.withUnsafeBytes { (bytes: UnsafePointer<_>) -> (String, Int)? in
```

Then the compiler complaint with,

```sh
Cannot convert value of type '(_) -> (String, Int)?' to expected argument type '(UnsafePointer<ContentType>) -> ResultType'
```

Plus my environment,
```sh
$ swift --version
Apple Swift version 3.0.2 (swiftlang-800.0.63 clang-800.0.42.1)
Target: x86_64-apple-macosx10.9

$ swiftformat --version 
swiftformat, version 0.23
```

**Comment by nicklockwood** [comment]
Hi, I believe this was fixed in version 0.23.1 (release yesterday). Can you let me know if not? Thanks.

**Comment by longkai** [comment]
Yes, it is fixed. I am sorry I ran the old version. Thanks.
